packages/react-components/react-charts-preview/src/components/LineChart/LineChart.tsx

```diff
@@ -966,7 +997,7 @@ export const LineChart: React.FunctionComponent<ILineChartProps> = React.forward
           .attr('y2', `${lineHeight - _yAxisScale(pointToHighlight.y)}`);
 
         setNearestCircleToHighlight(pointToHighlight);
-        setIsCalloutVisible(true);
+        updatePosition(mouseEvent.clientX, mouseEvent.clientY);
```

Comment:
dont need to set isPopverOpen here?

Reply:
yes isPopoverOpen is set within updatePosition